scripts/wattlab_parity_watch_ghcr.py

```
@@ -0,0 +1,139 @@
+                )
+            cfg_digest = man["config"]["digest"]
+            req = urllib.request.Request(
+                f"https://ghcr.io/v2/{repo}/blobs/{cfg_digest}",
+                headers={"Authorization": f"Bearer {tok}"},
+            )
+            with urllib.request.urlopen(req, timeout=60) as r:
+                cfg = json.load(r)
+            created = cfg.get("created") or ""
+            rows.append((created, tag, digest))
+        except Exception:
+            continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)